### ISSUE TITLE
[relay 13] environment.executeSubscription is not a function

### ISSUE THREAD

**Issue by TassiloSinghammerVolue** [facebook/relay#3757]
After upgrading to Relay 13 I'm getting the following runtime error:
environment.executeSubscription is not a function

I've seen that executeSubscription has been added with this PR, so I don't really understand why it can't find the function:
https://github.com/facebook/relay/commit/e11a33476b0958a858191769c39acc254e9e0ac3

**Comment by TassiloSinghammerVolue** [comment]
It also seems that the typings in RelayModernEnvironment.d.ts rsp. RelayStore.Types.d.ts are not updated yet. They don't contain executeSubscription either,

**Comment by daniloab** [comment]
hi, @TassiloSinghammerVolue can you place a print/text from the completely error message?

**Comment by TassiloSinghammerVolue** [comment]
Hi @daniloab It' just this message I'm getting. Whether in jest tests or when run in the browser.
The complete stack ist:

TypeError
environment.executeSubscription is not a function
Call Stack
 requestSubscription
  vendor.js:142307:25
 undefined
  vendor.js:128392:33
 invokePassiveEffectCreate
  vendor.js:117477:20
 HTMLUnknownElement.callCallback
  vendor.js:97940:14
 Object.invokeGuardedCallbackDev
  vendor.js:97989:16
 invokeGuardedCallback
  vendor.js:98051:31
 flushPassiveEffectsImpl
  vendor.js:117564:9
 unstable_runWithPriority
  vendor.js:186758:12
 runWithPriority$1
  vendor.js:105271:10
 flushPassiveEffects
  vendor.js:117437:14


requestSubscription is called from the useSubscription hook.

**Comment by artola** [comment]
@TassiloSinghammerVolue I just re-checked running a local `useSuscription` with v13 and I have no errors. I would suggest you to check the version of the installed packages and may be wipeout the webpack/babel cache.

```
react-relay 13.0.1
relay-runtime 13.0.1
```

Regarding the TS typings, they are not maintained in this project, else by other contributors in [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/relay-runtime), you can help to enhance them with a PR (or for a quick solution, just augment them in your project).

**Comment by TassiloSinghammerVolue** [comment]
@artola thanks for checking. We're using Nx mono repo, so it may have to do with so configuration there. But it's still strange.
I have cloned our repository freshly into a different folder, so there shouldn't be anything cached.
The versions are fine, 13.0.1 in package.json, package-lock-json, I even checked what's actually inside node_modules and it's also 13.0.1
Still getting this error.
Yeah I think DefinitelyTyped is one feature version behind. I will check if there is no PR waiting already. For me it's not an issue currently, I'm not using executeSubscription anywhere directly.
Did you you CRA?

**Comment by TassiloSinghammerVolue** [comment]
Hey, update!
It's working now. I did not have a direct dependency to relay-runtime.
Could it be that react-relay is referencing an older dependency of relay-runtime?
Anyway, thanks for helping me out!

**Comment by artola** [comment]
@TassiloSinghammerVolue Yes, this is my thought. In general, `react-relay` has dependency for `relay-runtime`, I bet that you got another (legacy) version when bundling the dependencies (I saw this problem in previous upgrades also).

I have several apps running v12, v13@latest and v13@main all without this problem. Also, check that in v13 you do not use anymore `relay-config` (it is integrated into `relay-compiler`).
